arch/arm/soc/atmel_sam/sam4s/soc_pinmap.h

```diff
@@ -17,13 +17,13 @@
 
```

Comment:
The include guard should be better defined as _ATMEL_SAM4S_SOC_PINMAP_H_ since the guard in soc.h file is also series specific. This was copied from SAME70 where it needs to be fixed too.